game.c

```diff
@@ -168,8 +186,14 @@ bool val_in_column(cell** board, int column, int row, int val){
  * if value is invalid it marks it with an asterisk */
 bool valid_check(cell** board, int column, int row, int val) {

```

Comment:
Missing return value the end of the function outside the conditions, should it be `return false;` ?